src/ensembl/src/shared/components/communication-framework/contact-us/ContactUs.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+
+import { SecondaryButton } from 'src/shared/components/button/Button';
+import ExternalLink from 'src/shared/components/external-link/ExternalLink';
+
+import { ReactComponent as BlogIcon } from 'static/img/home/blog.svg';
+import { ReactComponent as FacebookIcon } from 'static/img/home/facebook.svg';
+import { ReactComponent as TwitterIcon } from 'static/img/home/twitter.svg';
+
+import styles from './ContactUs.scss';
+
+const ContactUs = () => {
+  const [shouldShowForm, setShouldShowForm] = useState(false);
+
+  if (shouldShowForm) {
+    return <div>Will display the form</div>;
+  }
+
+  return (
+    <div className={styles.wrapper}>
+      <section>
+        <p>
+          Please contact us if you have a problem with the website or need help
+        </p>
+        <SecondaryButton onClick={() => setShouldShowForm(!shouldShowForm)}>
+          Contact us
+        </SecondaryButton>
+      </section>
+      <section className={styles.socialMediaInfo}>
+        <p>
+          Visit the blog for details of releases, workshops and other tidbits of
+          information about the Ensembl project.
+        </p>
+        <p>
+          Visit Facebook or Twitter for the latest breaking Ensembl news and
+          service status updates.
+        </p>
+        <p>
+          <a href="https://www.ensembl.info/">
+            <span className={styles.socialMediaLinkText}>Ensembl Blog</span>{' '}
+            <BlogIcon className={styles.icon} />{' '}
+          </a>
+          <a href="https://www.facebook.com/Ensembl.org/">
+            <FacebookIcon className={styles.icon} />{' '}
+          </a>
+          <a href="https://twitter.com/ensembl">
+            <TwitterIcon className={styles.icon} />
+          </a>
+        </p>
+      </section>
+      <section>
+        <h2>Online Help</h2>
+        <p>
+          Each app has a Help icon (top right) that will show context-sensitive
+          help for that view.
+        </p>
+        <p>All Help articles and videos can be found in the Help app.</p>
+      </section>
+      <section>
+        <h2>Mailing lists</h2>
+        <p> We have two public mailing lists: </p>
+        <dl>
```

Comment:
I remember we used `dl`, `dt` tags on the track panel but then we changed it later on. Should we continue to use it?

Reply:
We discussed it with Imran in an [earlier comment](https://rapid.ensembl.org/index.html).

There's nothing inherently wrong with `dl` etc in itself. HTML tags are used to convey meaning. The meaning of `dl` etc is to represent a list of descriptions. The prime example of such a list is a glossary, where a term (marked with `dt`) is explained in a description (marked with `dd`).

Imran felt that this:

<img src="https://user-images.githubusercontent.com/6834224/127845180-c527d498-22bb-4e9e-b0e9-3ec7e9b3801d.png" width="500" />

looks like such a description list, where "announce" and "dev" are the terms, and the paragraphs under them are their descriptions. It wouldn't have occurred to me to think about it this way, but I have to agree that it kind of does look like it. So I agreed that the `dl` etc. were used here as they are intended to be used.